battery-service/src/lib.rs

```diff
@@ -122,10 +128,24 @@ impl<
                     .await
                     .unwrap();
             }
+            BatteryMsgs::Oem(_) => (),
             _ => todo!(),
```

Comment:
With the additional `BatteryMsgs::Oem(_)` arm above, this catch-all is now unreachable.